ramls/item.json

```diff
@@ -296,6 +296,47 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
+      "type": "object",
+      "readonly": true,
```

Comment:
Can a parent object be `readonly` and its children not be? Based on this schema, I don't understand how these properties are to be used in mod-inventory-storage.

Reply:
done

Reply:
It was readonly by mistake